corehq/apps/ota/views.py

```diff
@@ -79,7 +79,6 @@
 @location_safe
 @handle_401_response
 @mobile_auth_or_formplayer
```

Comment:
not really related but why does the restore endpoint need `mobile_auth_or_formplayer`? Is it because SMS forms don't have user auth?

Reply:
yep, I believe that's the case.
https://github.com/dimagi/commcare-hq/blob/d5b9a30c60ed44d8e9ba1992c778cf7b002753ae/corehq/apps/domain/decorators.py#L361-L364
The restore endpoint is the only place that mechanism is used.